src/index.ts

```diff
@@ -188,15 +285,26 @@ export default class OneSignal {
         return deviceState;
     }
 
-    static setLanguage(language) {
+    /**
+     * Allows you to set the app defined language with the OneSignal SDK.
+     * @param  {string} language
+     * @returns void
+     */
+    static setLanguage(language: string): void {
         if (!isObjectNonNull(RNOneSignal)) return;
         
         RNOneSignal.setLanguage(language);
     }
 
     /* T A G S */
 
-    static sendTag(key, value) {
+    /**
+     * Tag a user based on an app event of your choosing so they can be targeted later via segments.
+     * @param  {string} key
+     * @param  {string} value
+     * @returns void
+     */
+    static sendTag(key: string, value: string | boolean): void {
```

Comment:
In the [iOS SDK](https://github.com/OneSignal/OneSignal-iOS-SDK/blob/fd06483b314f842c16e0a6f74dfb27a04dc7df66/iOS_SDK/OneSignalSDK/Source/OneSignal.m#L1353) and [Android SDK](https://github.com/OneSignal/OneSignal-Android-SDK/blob/54e4b0fe45bdb7168788d2072a98d573e5a3e599/OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java#L1813) and OneSignal documentation (https://documentation.onesignal.com/docs/data-tag-implementation#sendtag-method), value is `string`. 

I added `boolean` to the value type here because in the body of this method, we are letting users pass in a `boolean`.
```
        if (typeof value === "boolean") {
            value = value.toString();
        }
```
❓ Should I add `number` here as well and convert it `toString`. OR, remove the `boolean` type and only accept string values?

In Flutter, value is a `dynamic` type, and in Cordova, there is no type (except the implementation file says `string`).

Reply:
Done.. `tags` should only have `string` values